tasks/load_ocpinventory.yml

```diff
@@ -0,0 +1,27 @@
+# ocpinventory.json for MAC addresses
+---
+- name: Download ocpinventory.json
+  get_url:
+      url: "{{ alias.lab_url }}/cloud/{{ cloud_name }}_ocpinventory.json"
+      dest: ~/
+      mode: '0644'
+      force: yes
+  when: lab_name == "alias"
+
+- name: Download ocpinventory.json
+  get_url:
+      url: "{{ scale.lab_url }}/cloud/{{ cloud_name }}_ocpinventory.json"
+      dest: ~/
+      mode: '0644'
+      force: yes
+  when: lab_name == "scale"
+
+- name: set fact for ocpinventory file
+  set_fact:
+    ocpinventory_file: "~/{{ cloud_name }}_ocpinventory.json"
+
+- name: read ocpinventory file
+  set_fact:
+    ocpinventory_content: "{{ ocpinventory_content }}"
+  vars:
+    ocpinventory_content: "{{ lookup('file', '{{ ocpinventory_file }}') | from_json }}"
```

Comment:
Does
```
- name: read ocpinventory file
  set_fact:
    ocpinventory_content: "{{ lookup('file', '{{ ocpinventory_file }}') | from_json }}"
```
not work?
What is the purpose for setting both set_fact and vars?

Reply:
No specific reason, I will update this!

Reply:
updated!